packages/common-integration-tests/src/auth-client.ts

```diff
@@ -80,13 +101,16 @@ export function runAuthClientTests(
 
       expect(refreshSuccessRst.is_success);
 
-      expect(generateSuccessRst.expiresAt.epoch()).toEqual(
-        refreshSuccessRst.getExpiresAt().epoch()
-      );
+      const expiresAtDelta =
+        refreshSuccessRst.expiresAt.epoch() -
+        generateSuccessRst.expiresAt.epoch();
+
+      expect(expiresAtDelta).toBeGreaterThanOrEqual(delaySecondsBeforeRefresh);
```

Comment:
hrm @rlinehan I should probably also add a lessThanOrEqual assertion here

Reply:
did this in https://github.com/momentohq/client-sdk-nodejs/pull/515/commits/7240edfe018b4d380f56f1c8d91a2f15aead6d85